### DISCUSSION TITLE
Does this library have ever been used as a game engine?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/eepp#64] [General]
It advertised it's also a game engine. But so far I don't see any games written in it. I think it's more like an immediate mode gui like ImGui.

**Comment by SpartanJ** [comment]
> It advertised it's also a game engine. But so far I don't see any games written in it.

Yes, I've developed around 10 small games with it, but they are commercial and not distributed over the internet.

> I think it's more like an immediate mode gui like ImGui.

That would cover only the UI module, but eepp provides system, audio, graphics, network, scene and window modules, plus some extra modules for 2D maps and terminal. It's a larger system that integrates all the necessary components to build games and multimedia applications. It's similar to what Qt is but with a different focus.
